iptables/iptables.go

```diff
@@ -36,7 +36,7 @@ func (e *Error) ExitStatus() int {
 }
 
 func (e *Error) Error() string {
-	return fmt.Sprintf("exit status %v: %v", e.ExitStatus(), e.msg)
+	return fmt.Sprintf("iptables exit status %v: %v", e.ExitStatus(), e.msg)
```

Comment:
Minor: we can also run ip6tables here. Please just leave this out of your PR, we have https://github.com/coreos/go-iptables/pull/36/files in progress covering that.